lib/netsuite/configuration.rb

```diff
@@ -22,6 +25,42 @@ def connection(params={}, credentials={})
       }.update(params))
     end
 
+    def ssl_version=(_ssl_version)
+      attributes[:ssl_version] = _ssl_version
+    end
+
+    def ssl_version(_ssl_version = nil)
+      if _ssl_version
+        self.ssl_version = _ssl_version
+      else
+        attributes[:ssl_version]
+      end
+    end
+
+    def ssl_verify_mode=(verify_mode)
+      attributes[:ssl_verify_mode] = verify_mode
+    end
+
+    def ssl_verify_mode(verify_mode = nil)
+      if verify_mode
+        self.ssl_verify_mode = verify_mode
+      else
+        attributes[:verify_mode]
+      end
+    end
+
+    def ssl_ca_cert_file=(path)
+      attributes[:ssl_ca_cert_file] = path
+    end
+
+    def ssl_ca_cert_file(path = nil)
+      if path
+        self.ssl_ca_cert_file = path
+      else
+        attributes[:ssl_ca_cert_file]
+      end
+    end
+
     def api_version(version = nil)
```

Comment:
Why were these additional SSL configuration options needed? I'm hesitant to merge them in unless there is a strong general use case.